src/Servers/IIS/IIS/test/testassets/InProcessWebSite/Startup.cs

```diff
@@ -696,6 +696,29 @@ private async Task ShutdownStopAsyncWithCancelledToken(HttpContext ctx)
             await server.StopAsync(cts.Token);
         }
 
+        private async Task StackSize(HttpContext ctx)
+        {
+            // This would normally stackoverflow if we didn't increase the stack size per thread.
+            RecursiveFunction(10000);
+            await ctx.Response.WriteAsync("Hello World");
+        }
+
+        private async Task StackSizeLarge(HttpContext ctx)
+        {
+            // This would normally stackoverflow if we didn't increase the stack size per thread.
+            RecursiveFunction(30000);
+            await ctx.Response.WriteAsync("Hello World");
+        }
+
+        private void RecursiveFunction(int i)
+        {
+            if (i == 0)
```

Comment:
Psh, I feel like you should at least calculate Pi or some Fibonacci numbers or something ;P